### ISSUE TITLE
:focus-visible selector not supported in Safari but used across almost all of our focusable elements

### ISSUE THREAD

**Issue by sarayourfriend** [WordPress/openverse-frontend#723]
## Description

[`:focus-visible` is unsupported in any current version of Safari](https://caniuse.com/?search=focus-visible)

## Reproduction

1. Open non tech-preview Safari (the current version or an iPad or iPhone safari)
2. Focus the header buttons using the keyboard
3. Note the lack of focus styles on any button

## Expectation

Buttons should have focus styles in all browsers

## Environment

 - Browser: Safari

## Additional context

Found while testing https://github.com/WordPress/openverse-frontend/pull/695

## Resolution

- [x] 🙋 I would be interested in resolving this bug.
